console/console-init/ui/src/modules/iot-device-detail/components/ConnectionGateway/ConnectionGateway.tsx

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright 2020, EnMasse authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+
+import React from "react";
+import { Link } from "react-router-dom";
+import { useParams } from "react-router";
+import {
+  Card,
+  Flex,
+  FlexItem,
+  CardBody,
+  Title,
+  Text,
+  Popover,
+  Button,
+  CardTitle
+} from "@patternfly/react-core";
+import { OutlinedQuestionCircleIcon } from "@patternfly/react-icons";
+import { StyleSheet, css } from "aphrodite";
+
+export interface IConnectionGatewayProps {
+  deviceList?: string[];
+  gatewayGroups?: string[];
+}
+
+const styles = StyleSheet.create({
+  card_body: {
+    marginBottom: "1.25rem"
+  }
+});
+
+export const ConnectionGateway: React.FC<IConnectionGatewayProps> = ({
+  deviceList,
+  gatewayGroups
+}) => {
+  const { projectname, namespace } = useParams();
+
+  return (
+    <Card className={css(styles.card_body)}>
```

Comment:
Please divide this component in 3 smaller components for Connection-Gateways, Gateway-Devices and Gateway-Groups